rc/v.kak

```diff
@@ -0,0 +1,54 @@
+##
+## v.kak by Conscat
+##
+
+# https://vlang.io
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .+\.(v|vsh) %{
+    set-option buffer filetype v
+}
+
+# Initialization
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+hook global WinSetOption filetype=v %{
+    require-module v
+    add-highlighter window/v ref v
+}
+
+hook global WinSetOption filetype=(?!v).* %{
+    remove-highlighter window/v
+}
+
+hook global BufSetOption filetype=v %{
+    set-option buffer comment_line '//'
```

Comment:
There's an implicit dependency here, you better `decl` that option before using it.

Reply:
I've now addressed this, but I'm not completely sure that I did so correctly.

Reply:
It looks good, but I think you'll also have to declare/set `comment_block_begin` and  `comment_block_end` (defined as `(*` and `*)` on my system, for some reason).

Could you also use the long form of command name, i.e. `decl` → `declare-option`?